editors/vscode/src/common.ts

```diff
@@ -151,12 +151,39 @@ export class PropertiesViewProvider implements vscode.WebviewViewProvider {
     }
 
     refresh_view() {
+        const editor = vscode.window.activeTextEditor;
+        if (editor === null) {
+            this.update_view(
+                "NO EDITOR",
+                this.#current_uri,
+                this.#current_version,
+                this.#current_cursor_line,
+                this.#current_cursor_character,
+            );
+            return;
+        }
+        const doc = editor.document;
+        if (doc === null) {
+            this.update_view(
+                "NO DOCUMENT",
+                this.#current_uri,
+                this.#current_version,
+                this.#current_cursor_line,
+                this.#current_cursor_character,
+            );
```

Comment:
did you mean to return here?
```suggestion
            );
            return;
```